### DISCUSSION TITLE
Output the value of the bounding box

### DISCUSSION THREAD

**Discussion by Limingxing00** [open-mmlab/mmdetection#8999] [Q&A] [answered]
Suppose I input the image size WxH is 1920x1080, for the coordinate value range  of H axis is 0 to 1079 or 1 to 1080? The predicted bounding box I get contains points on the boundary, such as H=1080.0 .

**Comment by jbwang1997** [comment, marked as answer]
This document may help you.
https://mmdetection.readthedocs.io/en/latest/compatibility.html#coordinate-system

**Reply by Limingxing00** [reply]
Hi,

Thank you for the fast reply!

I see "The new coordinate system is consistent with [Detectron2](https://github.com/facebookresearch/detectron2/) and treats the center of the most left-top pixel as (0, 0) rather than the left-top corner of that pixel. Accordingly, the system interprets the coordinates in COCO bounding box and segmentation annotations as coordinates in range [0, width] or [0, height]."

But, why are the coordinates 0 and width both included in the range [0, width]? The length of the entire image is width.

<img width="213" alt="image" src="https://user-images.githubusercontent.com/23522700/195303654-b048d857-d4e4-4206-8603-b47a9341a6af.png">

**Reply by jbwang1997** [reply]
In my understand, `0` means the point is at the left of the first pixel and `width` means the point is at the right of the last pixel.

**Reply by Limingxing00** [reply]
thank you!
